README.md

```diff
@@ -1,81 +1,217 @@
 # druid
 
-## Data-oriented Rust User Interface Design toolkit
+## A data first Rust-native UI toolkit.
 
-Druid is a new Rust-native UI toolkit, still in early stages. Its main
-goal is performance, also aiming for small binary size and compile time,
+[![crates.io](https://meritbadge.herokuapp.com/druid)](https://crates.io/crates/druid)
+[![docs.rs](https://docs.rs/druid/badge.svg)](https://docs.rs/druid/)
+[![license](https://img.shields.io/crates/l/druid)](./LICENSE)
+
+TKTK REVISIT GOALS? ... Its main goal is performance, also aiming for small binary size and compile time,
 fast startup, and very easy build configuration (just `cargo run`).
 It currently supports Windows, macOS, and GNU/Linux.
 
-Raph gave a talk at the July 2018 SF Rust Meetup ([video][jul-2018-video],
-[slides][jul-2018-slides]) about the design. Traditional object-oriented
-designs have proved to be cumbersome.
+It is currently used as a basis for [Runebender], a font editor written in Rust.
 
-The layout protocol is inspired by Flutter. It is originally being developed
-primarily for use by [xi-win], but has the potential to be useful for other
-applications. Due to its focus on performance and the relative paucity of
-toolkit-provided components, it is more likely to be useful for games and
-music software than general-purpose GUI applications. It is currently used
-as the basis for [Synthesizer IO].
+TKTK features go here.
 
-### The druid-win-shell layer
+## Example
 
-The druid-win-shell layer is primarily an abstraction for displaying a window
-and receiving events. As such, it is an alternative to [winit].
+Here's a simple counter example app.
 
-Ideally all Windows-specific logic (including all uses of `unsafe`) are isolated
-to the druid-win-shell subcrate, and druid proper is cross platform and with
-no unsafe code. That is not entirely the case now.
+```rust
+use druid::widget::{Align, Button, Column, Label, Padding};
+use druid::{AppLauncher, LocalizedString, Widget, WindowDesc};
 
-## Evolution
+fn main() {
+    let main_window = WindowDesc::new(ui_builder);
+    let data = 0_u32;
+    AppLauncher::with_window(main_window)
+        .use_simple_logger()
+        .launch(data)
+        .expect("launch failed");
+}
 
-This crate is currently in early stages. More features will be built out as
-needed for use in the flagship apps, and this will inevitably lead to API
-instability. Features hoped-for soon include:
+fn ui_builder() -> impl Widget<u32> {
+    let text =
+        LocalizedString::new("hello-counter").with_arg("count", |data: &u32, _env| (*data).into());
```

Comment:
I'm torn here; I think it's cool to show that we're doling localization, but I also think this example is somewhat impenetrable. I might at least add a comment to the effect,

```rust
// The label text will be computed based dynamically based on the current locale and count
```